django/db/backends/base/features.py

```diff
@@ -152,14 +152,13 @@ class BaseDatabaseFeatures:
     # Can the backend introspect a TimeField, instead of a DateTimeField?
     can_introspect_time_field = True
 
-    # Some backends may not be able to differentiate BigAutoField or
-    # SmallAutoField from other fields such as AutoField.
-    introspected_big_auto_field_type = 'BigAutoField'
-    introspected_small_auto_field_type = 'SmallAutoField'
-
-    # Some backends may not be able to differentiate BooleanField from other
-    # fields such as IntegerField.
-    introspected_boolean_field_type = 'BooleanField'
+    # Map fields class which some backends may not be able to differentiate
+    # them from other fields to the type it's introspected as.
+    introspected_field_type = {
```

Comment:
I should have suggested introspected_field_types (plural, to match similar attributes in DatabaseOperations) rather than singular.